### PR TITLE
Account for different UAS versions

### PR DESCRIPTION
**Pull request by rnmitchell**
It came to my attention that newer versions of the UAS software (at least v2.5) have a different format for the Sample Report than earlier versions. Further, the newest version no longer reports some SNPs on the reverse strand but all SNPs are reported on the forward strand. This PR will address these changes in order to be able to process any version of the UAS Sample Report.

### CODE REVIEW

**Comment by rnmitchell** [comment]
This is ready for review @standage

File: lusSTR/tests/test_snps.py [hunk 1]
```diff
@@ -239,3 +239,23 @@ def test_snp_bins(tmp_path):
...
+        "-o",
+        "kin_v5",
+        "--input",
+        input_sample,
+        "--snps",
+        "--kintelligence",
+    ]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
+    all_arglist = ["snps", "all", "-w", str(tmp_path)]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(all_arglist))
```
Comment (standage) [inline lusSTR/tests/test_snps.py:261]: It looks like this test isn't checking the output in any way. It's just making sure the code runs without exploding? (I call these "smoke tests" and use them frequently. 😎)

File: lusSTR/wrappers/snps_format.py [hunk 2]
```diff
@@ -231,7 +256,10 @@ def create_row(df, j, sampleid, projectid, analysisid):
     uas_allele = df.loc[j, "Allele Name"]
     try:
         metadata = snp_marker_data[snpid]
-        forward_strand_allele = check_rev_comp(uas_allele, snpid, metadata)
+        if ver == "2.0":
+            forward_strand_allele = check_rev_comp(uas_allele, snpid, metadata)
+        else:
+            forward_strand_allele = uas_allele
```
Comment (standage) [inline lusSTR/wrappers/snps_format.py:262]: That makes sense based on your description.

**Comment by rnmitchell** [comment]
Ok **now** this is ready for review 😅

**Comment by standage** [comment]
> Ok **now** this is ready for review 😅

Boom!

**Pull request merged**